soc/arm/infineon_xmc/4xxx/soc.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2020 Linumiz
+ * Author: Parthiban Nallathambi <parthiban@linumiz.com>
+ *
+ */
+
+#include <kernel.h>
+#include <init.h>
+#include <soc.h>
+
+#define PMU_FLASH_WS		(0x3U)
+
+void z_platform_init(void)
+{
+	u32_t temp;
+
+	/* Enable unaligned memory access - SCB_CCR.UNALIGN_TRP = 0 */
+	SCB->CCR &= ~(SCB_CCR_UNALIGN_TRP_Msk);
```

Comment:
Why is this needed? Is it required temporary? AFAIK (maybe I am wrong though) this is not done for other cortex-m platforms. Must justify.

Reply:
Agree, it was temporary and shouldn't be here. Removed, thanks.